micropython/drivers/sensor/sht3x/sht3x.py

```diff
@@ -0,0 +1,26 @@
+# SHT3x driver for MicroPython on ESP8266/ESP32
+# MIT license; Copyright (c) 2022 WEMOS.CC
+
+import utime
+
+
+class SHT3X:
+    def __init__(self, i2c, address=0x45):
+        self.bus = i2c
+        self.slv_addr = address
+        self.buf = bytearray(6)
+
+    def measure(self):
+        self.bus.writeto(self.slv_addr, b"\x24\x00")
+        utime.sleep(1)
+        self.buf = self.bus.readfrom(self.slv_addr, 6)
+
+    def humidity(self):
```

Comment:
I wonder about adding a non-floating point version of these functions (to avoid float allocations), i.e.return an integer in 1/10ths of a degree / percent. (The point being that if you remove this float allocation and with the readfrom_into above, then the whole driver is non-allocating).

Reply:
Or perhaps just have only the versions that return tenths and the calling code can simply convert to float by doing `s.humidity() / 10.0' if necessary.